source/iostreams.tex

```diff
@@ -2633,7 +2633,7 @@
 the function signature
 \tcode{basic_ostream\& stream::operator\shl(ios_base\& (*)(ios_base\&))}
 to permit expressions of the form
-\tcode{cout \shl dec}
+\tcode{cout \shl{} dec}
```

Comment:
Yeah, I think the idea that anyone needs to prevent guillemets is archaic and pointless, and so we really should just get rid of those two macros.